scripts/deploy.sh

```diff
@@ -99,6 +99,8 @@ if [[ "${DEPLOY_AIO}" == "yes" ]]; then
   then
     rm /etc/openstack_deploy/conf.d/swift.yml
   fi
+  rm -f /etc/openstack_deploy/conf.d/aodh.yml
+  rm -f /etc/openstack_deploy/conf.d/ceilometer.yml
```

Comment:
Thanks for the update, these two lines look more along the lines of what I'd expect (that is, an optional service being removed from an AIO in the same way Swift might be). The difference between these and Swift is that we don't explicitly support them yet. When that time comes, they can be wrapped in a conditional.